PXB-3106 Document transition ZSTD to GA (8.0.34-29) (#151)
percona/pxb-docs: docs/take-streaming-backup.md

special `xbstream` format. After it finishes streaming all of the data files
to `STDOUT`, it stops xtrabackup and streams the saved log file too.

> When compression is enabled, *xtrabackup* compresses the output data, except for the meta and non-InnoDB files which are not compressed, using the specified compression algorithm. Read about the supported compression algorithms in [Create a compressed backup](create-compressed-backup.md) document.

Reviewer: ```suggestion
When compression is enabled, *xtrabackup* compresses the output data, except for the metadata, using the specified compression algorithm. Read about the supported compression algorithms in [Create a compressed backup](create-compressed-backup.md) document.
```

Reviewer: Non-InnoDB tables are also compressed:

```bash
 🐬 marcelo  📂 ~  ▶
 ╰▶ $ ls /tmp/backup/test/
tb1.MYD.zst  tb1.MYI.zst  tb1_377.sdi.zst
```

Author: Updated
